packages/contracts-bedrock/src/boba/BobaTuringCredit.sol

```diff
@@ -149,4 +149,28 @@ contract BobaTuringCredit {
 
         IERC20(turingToken).safeTransfer(owner, _withdrawAmount);
     }
+
+    // ----------------------------------------------------
+    // New for Anchorage
+
+    // Could make this configurable, but for now it's hard-coded to a predeploy address
```

Comment:
If we made this configurable, would it give us more flexibility for what degree of EVM vs. Application space the implementation lands -- or are we generally protected by the ability to upgrade the contract itself?

Reply:
@mmontour1306 Any thoughts about this?

Reply:
The BobaTuringCredit contract is only needed for legacy support, and at present I can't think of any cases where we would need to make the HCHelper address configurable. Even if we do move some of the implementation logic out of the Sequencer we would still want to have the Helper contract deployed to a known address.